Dockerfile

```diff
@@ -8,16 +8,21 @@ RUN apk update && \
 
 # Copy our source code into the container.
 WORKDIR /go/src/github.com/chihaya/chihaya
-ADD . /go/src/github.com/chihaya/chihaya
+COPY . /go/src/github.com/chihaya/chihaya
 
 # Install our golang dependencies and compile our binary.
 RUN glide install
-RUN go install github.com/chihaya/chihaya/cmd/chihaya
+RUN CGO_ENABLED=0 GOOS=linux go install github.com/chihaya/chihaya/cmd/chihaya
```

Comment:
I've not used docker before, ever, but doesn't it work on more than just linux?

Reply:
Nice catch! Removed, thanks.